src/travis.py

```diff
@@ -8,8 +8,8 @@
 import requests
 from google.cloud import kms_v1
 
-DATE_MECHANISM_FIRST_ENABLED = datetime(year=2019, month=7, day=15, tzinfo=timezone.utc)
-PANTSBUILD_PANTS_REPO_ID = 402860
+DATE_MECHANISM_FIRST_ENABLED = datetime(year=2020, month=1, day=22, tzinfo=timezone.utc)
```

Comment:
Not strictly necessary, but figure it's slightly safer this way. I chose yesterday so that builds from last evening can work.